samples/HttpsPolicySample/Startup.cs

```diff
@@ -21,7 +21,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddHttpsRedirection(options =>
             {
                 options.RedirectStatusCode = StatusCodes.Status301MovedPermanently;
-                options.TlsPort = 5001;
+                options.SSLPort = 5001;
```

Comment:
The correct term is Tls, please revert this change and avoid using SSL

Reply:
The same property is called SslPort at least two other places in the stack. Consistency is easier to understand. We don't actually know what protocol the underlying server is using, and most people don't know the difference anyways.

Reply:
https://github.com/search?l=C%23&q=org%3Aaspnet+SslPort&type=Code
SslPort: launchsettings.json, MvcOptions, RequireHttpsAttribute, AddRedirectToHttps.

Reply:
Two wrongs don’t make a right. Should we just call it https port moving forward? That way we use the scheme instead of the protocol.

MvcOptions and RequireHttpsAttribute are not recommended anymore. The port on mvcoptions is for development mostly.

Reply:
These are all primarily for development.

It doesn't merit a long debate. I appeal to @Eilon as the namer of arbitrary things.

Reply:
If we're creating new names, I think they should use the correct terminology, i.e. `HTTPS` and not `SSL`. Even though `TLS` might be correct right now, who knows what the future will hold. The term `HTTPS` should hopefully outlive all of those.

Reply:
(Unless, that is, something is actually referring to a specific lower-level protocol. But that doesn't appear to be the case here.)